internal/commands/remove_registry.go

```diff
@@ -0,0 +1,67 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+)
+
+func RemoveRegistry(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
+	var (
+		registryName string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "remove-registry <name>",
+		Args:  cobra.ExactArgs(1),
+		Short: "Remove registry",
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			registryName = args[0]
+
+			if registryName == config.OfficialRegistryName {
+				return errors.Errorf("%s is a reserved registry name, please provide a different registry",
+					style.Symbol(config.OfficialRegistryName))
+			}
```

Comment:
Are we sure this is the desired behavior? Users could want to remove it for security/compliance purposes, for instance

Reply:
The desired behavior is to always provide a functioning default registry by default, which is why we have this in place.  For the question on security/compliance, I'm not concerned at this point.  I'd rather get some customer feedback before making too many assumptions.